Src/TestSample.BroadInt.Tests/Users/User.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Users
+

```

Comment:
Like Narrow Integration tests have 2 sets of tests for grpc and REST (in the future: Kafka), I believe we will have to do the same with Broad Integration tests. 
If not, why?